pkg/osbuild/source.go

```diff
@@ -54,25 +75,55 @@ func (sources *Sources) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
-func GenSources(packages []rpmmd.PackageSpec, ostreeCommits []ostree.CommitSpec, inlineData []string, containers []container.Spec) (Sources, error) {
+func addPackagesCurl(sources Sources, packages []rpmmd.PackageSpec) error {
```

Comment:
Would it make sense to make `addPackagesCurl()` a method of the `Sources` type? Especially since it modifies it.

Reply:
Great, idea, I moved it